components/raftstore/src/store/snap.rs

```diff
@@ -691,7 +691,13 @@ impl Snapshot {
             let cf_stat = if plain_file_used(cf_file.cf) {
                 let key_mgr = self.mgr.encryption_key_manager.as_ref();
                 snap_io::build_plain_cf_file::<EK>(
-                    path, key_mgr, kv_snap, cf_file.cf, &begin_key, &end_key,
+                    path,
+                    key_mgr,
+                    kv_snap,
+                    cf_file.cf,
+                    &begin_key,
+                    &end_key,
+                    &self.mgr.limiter,
```

Comment:
Only lock cf will use plain cf. And lock cf is usually very small, like several bytes to several kilobytes.

Reply:
But technically plain cf can be configured for write cf as well right?
I think it's better to have a complete coverage. It does not hurt at least.